auth-web/src/routes/router.ts

```diff
@@ -63,9 +63,34 @@ export function getRoutes (): RouteConfig[] {
     {
       path: '/home',
       name: 'home',
-      component: getEnvHomeView(),
-      children: getEnvChildRoutes(),
-      meta: { showNavBar: !ConfigHelper.getLaunchFeatureFlag() }
+      component: HomeView,
+      children: [
+        {
+          path: '',
+          redirect: 'decide-business'
+        },
+        {
+          path: 'decide-business',
+          component: DecideBusinessView,
+          meta: { showNavBar: false }
```

Comment:
We might not need the `meta` data for the child routes if the `showNavBar` is false on the parent route. No biggie either way though, might be worth a test.